src/reducers/index.js

```diff
@@ -1,25 +1,39 @@
+import { createStore, combineReducers } from 'redux';
+import { reducer as formReducer ,values} from 'redux-form';
 import axios from 'axios';
-const reducer = (state = {}, action) => {
-    let response;
-    let signUpURL = `https://secure-refuge-14993.herokuapp.com/add_user?username=${action.uname}&password=${action.password}&role=${action.role}`;
-    switch (action.type) {
-       case 'USER_SIGNUP':
-        console.log('inside signup-->',action.uname, action.password, action.role);       
-        axios.post(signUpURL).then((res)=>{
-            //on success
-            console.log('response',res);
-            response = res;
-        }).catch((error)=>{
-            //on error
-            console.log(error);
-        });
-          return response;
+// action types
+const ADD_USER = "ADD_USER";
+const API_CALL_SUCCESS = "API_CALL_SUCCESS";
+const API_CALL_FAILURE = "API_CALL_FAILURE";
 
-       case 'SAGA_USER_SIGNUP':
-       return;
+// reducer with initial state
+const initialState = {};
 
-       default:
-          return state;
-     }
-  };
-  export default reducer;
\ No newline at end of file
+export function reducer(state = initialState, action) {
+  switch (action.type) {
+    case ADD_USER:
+    axios.post(`https://secure-refuge-14993.herokuapp.com/add_user?username=${values.username}&password=${values.password}&role=${values.role}`)

```

Comment:
create a config and set base url in that

Reply:
ok sir